app/accountsdesk/views.py

```diff
@@ -0,0 +1,145 @@
+from flask_login import login_required
+from flask_wtf import Form
+from wtforms import SelectField
+from operator import attrgetter
+from . import accountsdesk
+
+from kconfig import agileCalendar
+from kconfig import accountsdeskBookByName, labsBookByName
+from kernel.IssuesList import IssuesList, IssuesFactory
+from kernel.Desk import TrackerDeskReporter
+from kernel.DataBoard import Data
+from kernel.NM_AccountsDeskReporter import ADeck, DeskReporter, ChannelReporter
+
+
+
+class SelectForm(Form):
+    select = SelectField(u'Items')
+
+@accountsdesk.route("/")
+@accountsdesk.route("/overview")
+@login_required
+def overview():
+    return redirect(url_for('.requests'))
+
+
+@accountsdesk.route("/accounts/requests", methods=['GET', 'POST'])
+@login_required
+def requests():
+    refresh = request.args.get('refresh') if request.args.get('refresh') else False
+    form = SelectForm()
+    deskname = 'Accounts-Desk'
+    desk = accountsdeskBookByName[deskname]
+    options = [(n,item) for n,item in enumerate(desk.channels)]
+    if request.method == 'POST':
+        channelname = dict(options)[int(form.select.data)]
+        return redirect(url_for('.requestToChannel', channelname=channelname))
+    form.select.choices = options
+
+    if refresh:
+        data = ADeck(*Data.getAccountDeskRequests())
+        if data.source == 'store':
+            flash('Unable to get fresh data from JIRA server')
+        else:
+            reporter = DeskReporter(desk, data)
+    else:
+        data = ADeck(*Data.getFocusedAccountDeskRequest())
+        if data.source == 'store':
+            flash('Data from local storage obtained at {}'.format(data.timestamp))
+
+    reporter = DeskReporter.fromFile(desk)
+
+    return render_template('accountsdesk/requests.html',
+                           desk = desk,
+                           data = data,
+                           reporter = reporter,
+                           form = form,
+                           calendar=agileCalendar)
+
+
+@accountsdesk.route("/accounts/requests/<channelname>", methods=['GET', 'POST'])
+@login_required
+def requestFromChannel(channelname):
+    refresh = request.args.get('refresh') if request.args.get('refresh') else False
+    form = SelectForm()
+    deskname = 'Accounts-Desk'
+    desk = accountsdeskBookByName[deskname]
+    options = [(n,item) for n,item in enumerate(desk.channels)]
+    if request.method == 'POST':
+        channelname = dict(options)[int(form.select.data)]
+        return redirect(url_for('.requestFomChannel', channelname=channelname))
+    form.select.choices = options
+    channel = desk.channels[channelname]
+    if refresh:
+        data = ADeck(*Data.getAccountChannelRequests(channel))
+        if data.source == 'store':
+            flash('Unable to get fresh data from JIRA server')
+            reporter = ChannelReporter.fromFile(channel)
+        else:
+            if data.source == 'store':
```

Comment:
This code is part of the herency so I prefer in this version do not touch it so far. Maybe you can create an issue to reflect it (and some other things like I will do).